staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/validator.go

```diff
@@ -96,9 +99,23 @@ func (a customResourceValidator) ValidateUpdate(ctx context.Context, obj, old ru
 	return allErrs
 }
 
-// WarningsOnUpdate returns warnings for the given update.
-func (customResourceValidator) WarningsOnUpdate(ctx context.Context, obj, old runtime.Object) []string {
-	return nil
+var standardFinalizers = sets.NewString(
+	metav1.FinalizerKubernetes,
```

Comment:
this is already defined in k8s.io/api/core/v1, let's just reference it there

Reply:
It's defined in `k8s.io/kubernetes/pkg/apis/core/v1/helper` but not in`k8s.io/api/core/v1`. Maybe in `staging` directory we should not import `k8s.io/kubernetes/pkg/xxx`?

Reply:
Sorry，I misunderstood your meaning earlier.